### ISSUE TITLE
Mask data based on field name, rather than field data

### ISSUE THREAD

**Issue by Eliemer** [serilog-contrib/Serilog.Enrichers.Sensitive#7]
Is it possible to create a Masking Operator that masks specific fields regardless of their data? An example would be a `password` field

### Before:
```json
{
    "username": "MyCustomeName",
    "password": "MyPassword"
}
```
### After:
```json
{
    "username": "MyCustomeName",
    "password": "*** MASKED ***"
}
```

**Comment by sandermvanvliet** [comment]
@Eliemer thanks for this suggestion, it's implemented in ff4ac0c8f1a66e3a2d52c4619560e5e6e901b75b and should be on NuGet soon as version 1.1.0.0

**Comment by sandermvanvliet** [comment]
The new version is now available on NuGet
